packaging/mariner/azure-nvme-utils.spec.tmpl

```diff
@@ -20,16 +20,15 @@ Utility and udev rules to help identify Azure NVMe devices.
 %autosetup
 
 %build
-%cmake -DVERSION="%{version}-%{release}" .
-make %{?_smp_mflags}
+%cmake -DVERSION="%{version}-%{release}"
+%cmake_build
 
 %install
-make install DESTDIR=%{buildroot}
+%cmake_install
+
+%check
+%ctest
 
 %files
-/usr/sbin/azure-nvme-id
 /lib/udev/rules.d/80-azure-nvme.rules
```

Comment:
```suggestion
%{_libdir}/udev/rules.d/80-azure-nvme.rules
```

Reply:
libdir maps to /usr/lib64 on the Fedora image.... i don't see a macro for /usr/lib at https://docs.fedoraproject.org/en-US/packaging-guidelines/RPMMacros/

fixed other macros and updated this to %{_exec_prefix}/lib